src/v/utils/sstring_hash.h

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
this should be more like `absl_sstring_hash.h` 

since this imports absl

Reply:
valid point, will rename